### ISSUE TITLE
Typography font style setting

### ISSUE THREAD

**Issue by Anydog** [themeum/kirki#439]
Hi,
am I missing something (or my Win8 / Xampp system is doing it's work again ;) ) with typography field font-style controls ? Those are always in full (bold, italic, underline, strikethrough) no matter what I choose in config ( `array( 'bold', 'italic' ),` ). And, when I try to apply them all but bold style is not working ...
Shouldn't font-style lose "bold" setting (there's already a "font-weight" setting)?
Actually I don't need font-style setting (at the moment), so I commented it out, but it got my attention ...

**Comment by reshogun** [comment]
@Anydog  i noticed this also when i was playing with demo-theme, bold and strike are both somehow bugged so that if you ie, select strike and then italic bug overrides italic and font stays normal. 

Edit: @Anydog have tried to integrate kirki to your theme or are you running it as plugin? I am having problems with integration and/or using it as plugin.

**Comment by Anydog** [comment]
I have integrated into theme because, it's THEME customizer tool, and it's config applies only to particular theme. Also, I hate to add yet another plugin to required plugins list ;)
What problems do you have with using it as plugin ?

**Comment by reshogun** [comment]
Sorry, nothing with plugin anymore but I found with integration rather peculiar because as i see demo-theme working as it should, but when i try to integrate it to theme, it spawns those 404 not found error messages that were listed on the issue.  I am integrating kirki to theme as it is instructed 
![error5](https://cloud.githubusercontent.com/assets/3878710/10455692/6b30adde-71c5-11e5-89b3-acfb94fcb13c.png)

**Comment by Anydog** [comment]
Did you deactivate Kirki plugin ? And perhaps you have missing slash in assets url ... 
I know I had same issue, but I'm not sure what was the cause...

**Comment by reshogun** [comment]
```
/**
      * If Kirki is not already installed, use the included version
      */
    if ( ! class_exists( 'Kirki' ) ) {
    require get_template_directory() . '/inc/kirki/kirki.php';
    }
       if ( class_exists( 'Kirki' ) ) {
      code
       }
```

Basically this is what i am trying to get working. Kirki folder is inside inc folder. I am little lost  why this causes problems. Do you have your own version to show how did you included kirki on theme

**Comment by Anydog** [comment]
@aristath : any idea with this typo style ?
@reshogun : did you manage to solve theme/Kirki issue ?

**Comment by reshogun** [comment]
@Anydog  i am forced to develope everything as plugin, so no, i did not solve this problem. Any insight from your end ?
edit
@aristath  should number field type have any indicators what is up and down? This is little cosmetic thing but it seems to be missing

**Comment by Anydog** [comment]
@reshogun : sorry, man, I didn't "catch" any time to check this yet, I'm really into other issues with my theme right now. Did you find out the solution ?

**Comment by reshogun** [comment]
@Anydog  still having problems with it.

**Comment by aristath** [comment]
Hey guys,
The font-style property was a bit buggy so I disabled it for now.
Typography fields will have all other fields except this one.
This will allow us to push for a release sooner. The introduction of the typography field is important and the font-style issue is just a nuisance...
As soon as we release a stable version and iron-out all the important issues I'll get back to working on this one.

**Comment by aristath** [comment]
This has been fixed in the 2.2-beta bersion with the introduction of variants and the refactor of the typography fields.
